content/en/mobile_app_testing/_index.md

```diff
@@ -24,8 +23,14 @@ cascade:
     tags: ['mobile_testing']
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
+{{% site-region region="us3,us5,eu,gov,ap1" %}}
+<div class="alert alert-warning">
+  Mobile Application Testing is only supported for Datadog customers using the US1 site.
+</div>
+{{% /site-region %}}
+
+{{< callout url="#" header="false" btn_hidden="true" >}}
+  Mobile Application Testing is in limited availability.
```

Comment:
Can we update this to include a note about it being in US1 only so customers don't have to switch the Datadog site to see that? 

Something like:

Mobile Application Testing is in Limited Availability for customers using the US1 site only.